lib/byebug/interfaces/remote_interface.rb

```diff
@@ -16,25 +16,35 @@ def initialize(socket)
 
     def read_command(prompt)
       super("PROMPT #{prompt}")
+    rescue Errno::EPIPE, Errno::ECONNABORTED
+      "continue"
     end
 
     def confirm(prompt)
       super("CONFIRM #{prompt}")
+    rescue Errno::EPIPE, Errno::ECONNABORTED
+      false
+    end
+
+    def print(message)
+      super(message)
+    rescue Errno::EPIPE
```

Comment:
@MSP-Greg Should we rescue `Errno::ECONNABORTED` here and in `puts` as well? You were seeing that exception on Windows, but only under some versions if I recall correctly?

Reply:
I don't recall, I'll have to check.

Reply:
You mentioned it [here](https://github.com/deivid-rodriguez/byebug/pull/406#issuecomment-355767858). I think I'll add `Errno::ECONNABORTED` in the other methods as well then.

Re `exe/byebug`, feel free to suggest any changes you want in a separate PR!

Reply:
> You mentioned it here.

Sorry, rather distracted with `non code` things yesterday evening.

Travis has had large Linux & MacOS backlogs recently (separate occasions).  Hope they get it sorted out...